modules/effects/migrations/18_0_0-beta/index.ts

```diff
@@ -14,29 +14,47 @@ import {
   visitTSSourceFiles,
 } from '../../schematics-core';
 import { createRemoveChange } from '../../schematics-core/utility/change';
+import * as os from 'node:os';
 
 export function migrateConcatLatestFromImport(): Rule {
   return (tree: Tree, ctx: SchematicContext) => {
-    const changes: Change[] = [];
     addPackageToPackageJson(tree, 'dependencies', '@ngrx/operators', '^18.0.0');
 
     visitTSSourceFiles(tree, (sourceFile) => {
       const importDeclarations = new Array<ts.ImportDeclaration>();
+
       getImportDeclarations(sourceFile, importDeclarations);
 
-      const effectsImportsAndDeclaration = importDeclarations
+      const effectsImportsAndDeclarations = importDeclarations
         .map((effectsImportDeclaration) => {
           const effectsImports = getEffectsNamedBinding(
             effectsImportDeclaration
           );
           if (effectsImports) {
-            return { effectsImports, effectsImportDeclaration };
+            if (
+              effectsImports.elements.some(
+                (element) => element.name.getText() === 'concatLatestFrom'
+              )
+            ) {
+              return { effectsImports, effectsImportDeclaration };
+            }
+            return undefined;
           } else {
             return undefined;
           }
         })
-        .find(Boolean);
+        .filter(Boolean);
+
+      if (effectsImportsAndDeclarations.length === 0) {
+        return;
+      } else if (effectsImportsAndDeclarations.length > 1) {
+        ctx.logger.warn(
+          '[@ngrx/effects] Skipping because of multiple `concatLatestFrom` imports'
```

Comment:
Do we have to log this?
Or maybe change this to an info message because no action is required here.

Reply:
Changed it to info. Maybe it is good to keep it so that devs are aware of it.